infrastructure/aat.tfvars

```diff
@@ -2,4 +2,6 @@ vault_section = "preprod"
 scheduling_enabled = "true"
 file_cleanup_enabled = "true"
 
+encyption_enabled = "true"
```

Comment:
@rkondratowicz just noticed it is misspelled

Reply:
well spotted!
It's misspelled everywhere though (see https://github.com/hmcts/send-letter-service/blob/master/infrastructure/variables.tf#L64) so this should work.
I'll fix the spelling everywhere separately.